packages/common/src/bag3d/common/utils/files.py

```diff
@@ -101,7 +101,9 @@ def get_export_tile_ids() -> Sequence[str]:
 
     env = os.getenv("DAGSTER_ENVIRONMENT", "test")
     if env == "test":
-        root_dir = Path(os.getenv("BAG3D_FILESTORE")) / "reconstruction_input"
+        root_dir = (
+            Path(os.getenv("BAG3D_FILESTORE")) / "integration_party_walls/file_store"
```

Comment:
Why did you change this? Why set to `"integration_party_walls/file_store"`?

Reply:
It is not nested, for each integration test I have a filestore and a filestore_fastssd, so that it is clear (like we havei n production)